gm4_enchantment_extractors/data/enchantment_extractors/functions/extract_failed.mcfunction

```diff
@@ -1,6 +1,9 @@
-execute if score extract_chance gm4_ench_random matches ..80 run particle explosion ~ ~1.7 ~ 0.2 0.2 0.2 0.05 25 force @a[distance=..10]
-execute if score extract_chance gm4_ench_random matches ..80 run playsound item.shield.break block @a[distance=..8] ~ ~ ~ 0.6 0.3
+scoreboard players add fail_type gm4_ench_random 27
```

Comment:
The issue with using the 27-100 randomization method here is that this whole function only runs when something fails, so failures would always follow the same pattern. 1.8 used the rotation of the item because it was inherently random and isn't controllable by the player except by trial and error. The randomization for the enchantment success can depend on the 27-100 method since 27 gets added constantly, and the score's represents a more fully random number.

Reply:
I'm using a different fake player here. There would be a 80% chance of no effect, and 10% each for both vexes.

Reply:
@SpecialBuilder32 Is this still an issue? It seems that this review was based before I made the "Separate fail type and extract chance randomization" [commit](https://github.com/Gamemode4Dev/GM4_Datapacks/pull/9/commits/ec71742b4f3b988cbab63e94d94374cee645b594)

Reply:
I need to take a look at how it behaves in-game. Just from looking at the functions it looks like it might be but I am unsure. My current concern is that the randomizer won't run enough for the value to be sufficiently random.